tests/unittests/data/test_accessor.py

```diff
@@ -0,0 +1,123 @@
+"""Test :py:class:`pycompwa.data.PwaAccessor`."""
+
+
+from math import isclose
+from os.path import dirname, realpath
+
+import pandas as pd
+
+import pytest
+
+from pycompwa.data import _labels, append, exception
+
+
+SCRIPT_DIR = dirname(realpath(__file__))
+
+
+def import_test_frame(weights: bool = False):
+    """Import the pickled frame."""
+    if weights:
+        import_file = f'{SCRIPT_DIR}/files/pwa_frame_weights.pkl'
+    else:
+        import_file = f'{SCRIPT_DIR}/files/pwa_frame_noweights.pkl'
+    return pd.read_pickle(import_file)
```

Comment:
Hmm, I don't like using binary data for tests. Because its not clear what the structure and content of the data is...(etc)

Reply:
https://github.com/ComPWA/pycompwa/commit/e3b93d7c5e03be4c8a155c7dfa69869afa0d0dc8 :ok_hand: